src/main/java/io/appium/java_client/pagefactory/AppiumFieldDecorator.java

```diff
@@ -73,9 +73,9 @@
     private final HasSessionDetails hasSessionDetails;
 
 
-    public AppiumFieldDecorator(SearchContext context, long implicitlyWaitTimeOut,
+    public AppiumFieldDecorator(SearchContext context, long timeOut,
```

Comment:
-> timeout

Reply:
There are 32 places that use `timeOut` vs none with `timeout`. This PR should keep it consistent, though a later commit to change all usages is probably appropriate.

Reply:
Why can't we change all the instances?

Reply:
We can/should, just didn't want to deal with those changes in this PR/release. :-)